tests/functional/test_default_sender_context_manager.py

```diff
@@ -21,7 +21,11 @@ def test_default_sender_test_account(solidity_contract_instance, owner, test_acc
 
 
 def test_default_sender_account(
-    solidity_contract_container, networks_connected_to_tester, accounts, keyfile_account
+    eth_tester_provider,
+    solidity_contract_container,
+    networks_connected_to_tester,
```

Comment:
`networks_connected_to_tester` already should be using that, so im guessing this wont do it